presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfig.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.rubix;
+
+import io.airlift.configuration.Config;
+
+public class RubixConfig
+{
+    boolean rubixEnabled;
+    boolean parallelWarmupEnabled;
```

Comment:
I will be good to have `parallelWarmupEnabled` enabled by default

Reply:
done